### PR TITLE
[WIP] Add func-tests into operator image

### PR DESCRIPTION
**Pull request by erkanerol**
Signed-off-by: Erkan Erol <eerol@redhat.com>

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,7 +25,7 @@ sanity:
 	./hack/build-manifests.sh
 	git difftool -y --trust-exit-code --extcmd=./hack/diff-csv.sh
 
-build: build-operator build-csv-merger build-webhook
+build: build-operator build-csv-merger build-webhook build-functest
```
Comment (nunnatsa) [inline Makefile:28]: Please remove `build-webhook`. Not in use here.

Comment (erkanerol) [reply Makefile]: Since we call `make build` in sanity checks as well, I reverted this part and call the targets in Dockerfile selectively.

**Comment by erkanerol** [comment]
Question: Why don't we vendor files for func-tests? While adding the test binary into the operator image, should not we vendor dependencies of tests as well?  @tiraboschi  @nunnatsa

**Comment by erkanerol** [comment]
We decided to discuss this approach again and to find another way. Closing...

**Pull request closed without merging**